### ISSUE TITLE
Refetched query not updating "loading" prop

### ISSUE THREAD

**Issue by tnrich** [apollographql/react-apollo#1347]
Hey there,

I've got a component that is wrapped with an apollo query. I use react-apollo's "refetchQueries" to cause that query to update itself (after a mutation is run by a separate component). The new data is getting fetched, but unfortunately, the "loading" prop for that mutation does not appear to be getting changed. Thus I am not able to get a spinner to show over the component while it's updated data is getting loaded.

Is this how refetch queries is set up to work?

Is there a way to get the components interested in the query to be made aware that a fresh query is being made?

Hope that all makes sense.
Thanks!

**Comment by jbaxleyiii** [comment]
@tnrich would you be able to create a small reproduction showing this?

**Comment by t0x1c123** [comment]
Yeah same issue here

**Comment by cannibal** [comment]
Im experiencing the same issue, get the loading container as per the recipe in apollo docs for initial load - this does not appear on pagination queries using fetchMore because 'loading' variable is not updated.

**Comment by djyde** [comment]
how is it going?

**Comment by tknickman** [comment]
I just ran into this issue as well. I tried to proxy into the loading state with `notifyOnNetworkStatusChange` and `networkStatus`, but the `networkStatus` is `7` (indicating the data is [ready](https://www.apollographql.com/docs/react/api/react-apollo.html#graphql-query-data-networkStatus)) even when the query is being refetched from a mutation. 

I'm getting around this for now by passing `refetch` from the query I need to reload down and calling that in the Mutation `onComplete`. 

This is working fine, but `refetchQueries` would be a much more elegant solution (with a lot less prop drilling!)
